api/tag/models/tag.settings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "kind": "collectionType",
+  "collectionName": "tags",
+  "info": {
+    "name": "Tag"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true
+  },
+  "attributes": {
+    "tagName": {
+      "type": "string"
+    },
+    "courses": {
+      "collection": "course",
+      "via": "tags",
+      "dominant": true
+    },
+    "tadSlag": {
```

Comment:
I think its a typo here in this attribute name. I am assuming `tagSlug`?

Reply:
Oh, silly me :)
Fixed